packages/astro/src/core/build/generate.ts

```diff
@@ -1,7 +1,9 @@
 import * as colors from 'kleur/colors';
 import { bgGreen, black, cyan, dim, green, magenta } from 'kleur/colors';
 import fs from 'node:fs';
+import OS from 'node:os';
```

Comment:
nit: I find it a bit odd that this is uppercase 😄

Reply:
I bet @bluwy won't be sleeping at night 👀